### PR TITLE
fix(DB/Quest): Restricts multiple quests to blood elves

### LINKED ISSUE #7534: [Quest] Report to Splintertree Post incorrectly available to non Blood Elves
### What client do you play on?

enUS

### Faction

- [ ] Alliance
- [X] Horde

### Content Phase:

- [ ] Generic
- [X] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59

### Current Behaviour
Original report: https://github.com/chromiecraft/chromiecraft/issues/1517

Report to Splintertree Post quest 8 (https://wowgaming.altervista.org/aowow/?quest=9428) is available to non blood elf races.

### Expected Blizzlike Behaviour

The quest should only be Available to Blood Elves.

### Source

https://tbc.wowhead.com/quest=9428/report-to-splintertree-post

https://wowpedia.fandom.com/wiki/Report_to_Splintertree_Post

https://wotlk.evowow.com/?quest=9428

### Steps to reproduce the problem

1. Be level 18+ Horde and not a blood elf race
2. .go c 6506
3. See quest Available

### Extra Notes

_No response_

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/1f44732417968953c965e392c373cc453f79cf02

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-custom-corldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

### LINKED ISSUE #7535: Quest "Delivery to the Sepulcher" Incorrectly Available To Non-Blood Elf Characters
### What client do you play on?

enUS

### Faction

- [ ] Alliance
- [X] Horde

### Content Phase:

- [ ] Generic
- [X] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59

### Current Behaviour
Original report: https://github.com/chromiecraft/chromiecraft/issues/1519

The quest Delivery to the Sepulcher (https://wowgaming.altervista.org/aowow/?quest=9189) is available to non Blood Elf Characters.

Tester: confirmed, quest ID 9189 has the AllowableRaces field set to 690 = all Horde races, when it should be be 512 for belfs.

### Expected Blizzlike Behaviour

The quest should only be available to Blood Elf Characters.

### Source

https://tbc.wowhead.com/quest=9189/delivery-to-the-sepulcher

### Steps to reproduce the problem

1. Be 18+ Horde and not Blood Elf
2. Have Journey to undercity quest completed (https://wowgaming.altervista.org/aowow/?quest=9180)
3. Go to Ambassador Sunsorrow (https://wowgaming.altervista.org/aowow/?npc=16287)  .go c 32054
4. See the quest available

### Extra Notes

_No response_

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/1f44732417968953c965e392c373cc453f79cf02

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-custom-corldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

### LINKED ISSUE #7537: Quest "Report to Tarren Mill" Incorrectly Available To Non-Blood Elf Characters
### What client do you play on?

enUS

### Faction

- [ ] Alliance
- [X] Horde

### Content Phase:

- [ ] Generic
- [X] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59

### Current Behaviour
Original reportL https://github.com/chromiecraft/chromiecraft/issues/1518

The quest Report to Tarren Mill (https://wowgaming.altervista.org/aowow/?quest=9425) can be accepted and completed by non Blood Elf races.

Tester: confirmed quest 9425 can be accepted by a non-belf. Checked the AllowableRaces field and it's set to 0, which either means everyone or nobody (presumably the former since it can be accepted) and is incorrect either way.

### Expected Blizzlike Behaviour

The quest should only be available to Blood Elf characters.

### Source

https://tbc.wowhead.com/quest=9425/report-to-tarren-mill

### Steps to reproduce the problem

1. Be level 18+ Horde and not a BLood Elf
2. Go to Ambassador Sunsorrow (https://wowgaming.altervista.org/aowow/?npc=16287) - .go c 32054
3. See the quest available

### Extra Notes

_No response_

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/1f44732417968953c965e392c373cc453f79cf02

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-custom-corldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

### PR DESCRIPTION
**Pull request by jamadaha**
## Changes Proposed:
-  Restrict quest Report to Splintertree Post(ID 9428), Delivery to the Sepulcher(ID 9189) and Report to Tarren Mill(ID 9425) to only allow blood elfs

## Issues Addressed:

- Closes #7534
- Closes https://github.com/chromiecraft/chromiecraft/issues/1517
- Closes #7535 
- Closes https://github.com/chromiecraft/chromiecraft/issues/1519
- Closes #7537 
- Closes https://github.com/chromiecraft/chromiecraft/issues/1518

## SOURCE:

Before: 
Orc
image

After:
Orc
image
Blood Elf
image

## Tests Performed:

- Done test as described below. (Only for Report to Splintertree Post, checked Keira for the others)

## How to Test the Changes:

1. Do the following steps with a blood elf and a non blood elf - Both level 18+
2. .go c 6506/32054 
3. See if quest available 
4. For quest Delivery to the Sepulcher you need to finish quest 9180 first

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1629809640673167100.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1629809640673167100');
+
+# Restrict quest Report to Splintertree Post(ID 9428) to only allow blood elfs
+UPDATE `quest_template` SET `AllowableRaces` = 512 WHERE (`ID` = 9428);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1629809640673167100.sql:4]: newline

Comment (jamadaha) [reply data/sql/updates/pending_db_world/rev_1629809640673167100.sql]: fixed

**Comment by Azcobu** [comment]
While you're fixing this, do you want to add quests 9425 (for https://github.com/azerothcore/azerothcore-wotlk/issues/7537) and 9189 (for https://github.com/azerothcore/azerothcore-wotlk/issues/7535)? Don't have to of course, but thought it might save making a new PR.

**Comment by jamadaha** [comment]
Sure

**Comment by Azcobu** [comment]
I'd suggest using `ID` IN (9428, 9189, 9425); which saves you a couple of lines.

Thanks for that, I'll go test.

**Comment by Azcobu** [comment]
Tested, all looks good:

With non-belf these quests are not available:
WoWScrnShot_082421_230014
WoWScrnShot_082421_230034

With a belf they are:
WoWScrnShot_082421_230122
WoWScrnShot_082421_230436

**Review by LukasVolgger — APPROVED** [review]

**Pull request merged**